File: Practice/21/Python/21. Индекс массы тела.py
def bmi(weight: float, height: float):
    return (weight/(height*height))

def print_bmi(bmi: float):
    if (bmi < 18.5): print("Underweight\n")
    elif ((bmi >= 18.5) and (bmi < 25.0)): print ("Normal weight\n")
    elif ((bmi >= 25.0) and (bmi < 30.0)): print ("Overweight\n")
    elif (bmi >= 30.0): print("Obesity\n")

a,b=input().split()
a=int(a)
b=int(b)
b1=b/100
print_bmi(bmi(a,b1))
stop=input()#это для того, чтобы программа останавливалась

File: Practice/21/C++/21. Индекс массы тела/21. Индекс массы тела/21. Индекс массы тела.cpp
#include <iostream>
#include <cstdlib>
using namespace std;

double BMI(double weight, double height) {return (weight / (height * height));}
void printBMI(double BMI) {
    if (BMI < 18.5) cout << "Underweight\n";
    else if ((BMI >= 18.5) && (BMI < 25.0)) cout << "Normal weight\n";
    else if ((BMI >= 25.0) && (BMI < 30.0)) cout << "Overweight\n";
    else if (BMI >= 30.0) cout << "Obesity\n";
}
int main()
{
    double a,b,b1;
    cin >> a >> b;
    b1 = b / 100;
    double x = BMI(a,b1);
    printBMI(x);

    system("pause");
    return 0;
}